provisionerd/runner/runner.go

```diff
@@ -232,6 +232,10 @@ func (r *Runner) Run() {
 		err := r.sender.CompleteJob(ctx, r.completedJob)
 		if err != nil {
 			r.logger.Error(ctx, "sending CompletedJob failed", slog.Error(err))
+			err = r.sender.FailJob(ctx, r.failedJobf("internal provisionerserver error"))
+			if err != nil {
+				r.logger.Error(ctx, "sending FailJob failed (while CompletedJob)", slog.Error(err))
```

Comment:
The message here is difficult to parse, and yet even trickier to word in an informative yet succinct way.

"Failed to mark the job we previously failed to mark as completed as failed" just doesn't have the same ring to it.

If we fail to both 
a) mark the job as completed, and
b) fail to mark the completed job as failed

then isn't it likely that we are in an unrecoverable state?

Reply:
Per discussion: we can't bail out here because we need to close the loop. I don't have a better concrete suggestion for the log message, so resolving.